src/components/structures/LoggedInView.js

```diff
@@ -184,6 +184,13 @@ export default React.createClass({
                     handled = true;
                 }
                 break;
+            case KeyCode.KEY_K:
+                if (ev.ctrlKey && !ev.shiftKey && !ev.altKey && !ev.metaKey) {
```

Comment:
It should be cmd on Mac, see original code: https://github.com/vector-im/riot-web/pull/3933/files#diff-dc0dba0f86cd6bdf63884cc69d33431dR92